tests/unit/common/components/bradcrumbs.spec.js

```diff
@@ -0,0 +1,48 @@
+import { shallowMount, createLocalVue, RouterLinkStub } from "@vue/test-utils"
+import Breadcrumbs from "@/common/components/Breadcrumbs"
+import VueRouter from "vue-router"
+
+describe("Breadcrumbs component", () => {
+
+  const stubs = {
+    RouterLink: RouterLinkStub
+  }
+  
+  it("Should render breadcrumbs ", () => {
+    const router = new VueRouter()
+    const localVue = createLocalVue()
+    localVue.use(VueRouter)
+    const breadcrumb = shallowMount(Breadcrumbs, {
+      localVue,
+      router,
+      stubs
+    })
+
+    expect(breadcrumb.exists()).toBe(true)
+  })
+
+  
+  it("Watcher should return array of strings", () => {
+    const spy = jest.spyOn(Breadcrumbs.watch.$route, "handler")
+    expect(spy).toHaveReturned
+  })
+
+  it("Should render text with link title", () => {
+    const router = new VueRouter()
+    const localVue = createLocalVue()
```

Comment:
can you make this local Vue global refer to [this](tests/unit/common/components/dropdown.spec.js) file, so we can make a code more consistent

Reply:
I tried to make it global first , but I got error "Maximum call stack size exceeded" . That's why I make it locally as per article I found. Do you have any other suggestion how to handle the error ?

Reply:
I think you have to leave my comment for now

Reply:
Hi mas @bukandicki . As you already approved the PR . I'll resolve this conversation yaa . thank you